src/QMCHamiltonians/LocalECPotential.h

```diff
@@ -35,9 +35,6 @@ struct LocalECPotential : public OperatorBase
   typedef OneDimGridBase<RealType> GridType;
   typedef OneDimCubicSpline<RealType> RadialPotentialType;
 
-
-  typedef DistanceTableData::RowContainer RowContainerType;
```

Comment:
its nice that the use of auto removes the need for this typedef

Reply:
I deleted more places.